### ISSUE TITLE
about noise_whiting

### ISSUE THREAD

**Issue by CanCanZeng** [Wyp20/panorama-based-OpenVINS#5]
@Wyp20 Hello, I am very interested in your work. I tried running your algorithm and found an additional **noise_whiting** parameter. What does this parameter do? I found a bug when using it, which is that **uv_norm** is 2D, but you accessed its third dimension.
https://github.com/Wyp20/panorama-based-OpenVINS/blob/4fb6d1ea95ac3f9e48adf063b1a5314b2203a684/ov_msckf/src/update/UpdaterHelper.cpp#L350

**Comment by Wyp20** [comment]
You are right, **uv_m(2)** should be changed to **uv_m(1)**. Because there is severe distortion at the upper and lower edges of the panoramic photo, when this parameter is enabled, the feature points in the severely distorted area will be given a lower weight.

**Comment by CanCanZeng** [comment]
Thank you for your prompt reply. I found during the running that although points with FOV greater than 180 were used for optical flow tracking, they were not converted to SLAM points. Is this normal?

**Comment by Wyp20** [comment]
Usually they will be successfully converted
